dev-python/spyder/spyder-4.1.3.ebuild

```diff
@@ -0,0 +1,156 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
Python 3.8?

Reply:
Same as above, can't add py3_8 because I need py3_8 on pyls and spyder-kernels first (and some other deps, ipython and such)